unit-tests/dds/options-server.py

```diff
@@ -0,0 +1,71 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+import pyrealdds as dds
+from rspy import log, test
+
+dds.debug( True, log.nested )
+
+
+participant = dds.participant()
+participant.init( 123, "options-server" )
+
+
+def test_no_options():
```

Comment:
We already test this elsewhere, no?

Reply:
We also have other tests with device that have no options but only as something that you must initialize so you set it as empty. I wanted to concentrate all options related tests to a single file. Including the no_options test explicitly.